src/suites/cts/gpu_test.ts

```diff
@@ -153,4 +191,48 @@ export class GPUTest extends Fixture {
     }
     return undefined;
   }
+
+  checkBufferWithRowPitch(
+    actual: Uint8Array,
+    exp: Uint8Array,
+    width: number,
+    height: number,
+    rowPitch: number,
+    bytesPerPixel: number
+  ): string | undefined {
+    const size = exp.byteLength;
+    const lines = [];
+    let failedPixels = 0;
+    for (let i = 0; i < height; ++i) {
+      for (let j = 0; j < width * bytesPerPixel; ++j) {
+        const indexExp = j + width * i * bytesPerPixel;
```

Comment:
nit: give "width * bytesPerPixel" a name and use it on both lines

Reply:
Done.